dev/automated_tests/flutter_test/print_user_created_ancestor_expectation.txt

```diff
@@ -0,0 +1,27 @@
+<<skip until matching line>>
+══╡ EXCEPTION CAUGHT BY WIDGETS LIBRARY ╞═══════════════════════════════════════════════════════════
+The following assertion was thrown building
+RawGestureDetector-\[LabeledGlobalKey<RawGestureDetectorState>#.+\]\(state:
+RawGestureDetectorState#.+\(gestures: <none>, behavior: opaque\)\):
+'package:flutter\/src\/painting\/basic_types\.dart': Failed assertion: line 223 pos 10: 'textDirection
+!= null': is not true\.
+
+Either the assertion indicates an error in the framework itself, or we should provide substantially
+more information in this error message to help you determine and fix the underlying cause\.
+In either case, please report this assertion by filing a bug on GitHub:
+  https:\/\/github\.com\/flutter\/flutter\/issues\/new\?template=BUG\.md
+
+User created ancestor of the error causing widget was:
+
+CustomScrollView
```

Comment:
in the code the widget name and location are on the same line. Which option do you prefer? I added some comments indicating how to make both ways work.

Reply:
It is intended to be the same line, but the line is too long in this case that split into two